modules/Movecraft/src/main/resources/types/SubAirship.craft

```diff
@@ -137,32 +113,31 @@ halfSpeedUnderwater: true
 sinkPercent: 99.0
 overallSinkPercent: 60.0
 moveblocks:
-    152: # redstone represents engines
+    redstone_block: # redstone represents engines
         - 1.0
         - 100.0
-flyblocks:
-    42: # iron represents ballast tanks
+    iron_block: # iron represents ballast tanks
```

Comment:
Iron blocks are removed here, again to bring it into line with AP mechanics, but they aren't added to the crafts move-blocks

Reply:
I don't know why but it is there, you can check it on my actual repo and it is there but it doesn't display here for me either